catroid/src/main/java/org/catrobat/catroid/content/Look.java

```diff
@@ -257,8 +258,8 @@ public void setLookData(LookData lookData) {
 		imageChanged = true;
 	}
 
-	public boolean getAllActionsAreFinished() {
-		return scheduler.getAllActionsFinished();
+	public boolean getAllThreadsAreFinished() {
+		return scheduler.getAllThreadsFinished();
```

Comment:
please rename to areAllThreadsFinished() or haveAllThreadsFinished() or something like this.